### ISSUE TITLE
[popularity] a logo for ccls

### ISSUE THREAD

**Issue by findNextStep** [MaskRay/ccls#628]
now clangd has a logo in vscode extension
![image](https://user-images.githubusercontent.com/26270464/80684412-37264980-8af8-11ea-9f17-3587156db09b.png)
I think we need a logo for all ccls project (like vscode-ccls emacs-ccls) , so that the extension will not like a Pirate version 🤣
![image](https://user-images.githubusercontent.com/26270464/80684958-101c4780-8af9-11ea-8dbd-43bb52c6a9ad.png)

**Comment by aravind2612krishna** [comment]
![ccls](https://user-images.githubusercontent.com/3424615/85939701-5e9d6500-b935-11ea-86be-09ee6755eea8.png =128x128)

Here's an amateur attempt 😅

**Comment by aravind2612krishna** [comment]
More amateur attempts:
2. <img src="https://user-images.githubusercontent.com/3424615/85942835-91515880-b949-11ea-94ba-a8674812280b.png" width="128">
3. <img src="https://user-images.githubusercontent.com/3424615/85942837-931b1c00-b949-11ea-9e4b-fd495939b971.png" width="128">

**Comment by myrgy** [comment]
I like the 2 one.

**Comment by FederAndInk** [comment]
Maybe we could try with other colors 
I like 2 too

**Comment by Trass3r** [comment]
3 is a bit better I think.

**Comment by findNextStep** [comment]
Option 2 looks great, but it may better if LS is more square

**Comment by aravind2612krishna** [comment]
<img src="https://user-images.githubusercontent.com/3424615/85948678-cbcdec00-b96f-11ea-8b80-57464171f75e.png" width="128">
<img src="https://user-images.githubusercontent.com/3424615/85948679-ccff1900-b96f-11ea-83d9-3de9ceffdcfb.png" width="128">
 
I suck at color 😅 
Also, if anyone else has a better design, please do post 😄

**Comment by mys721tx** [comment]
An attempt at color coordination. 

![ccls](https://user-images.githubusercontent.com/561975/85954884-8b4e7e00-b92f-11ea-8323-4e33dbd6cc14.png)

**Comment by htfy96** [comment]
My $0.02 on palette:
- Two colors at most - IMO gradient doesn't play well with VSCode's design
- The official [C++ logo](https://upload.wikimedia.org/wikipedia/commons/thumb/1/18/ISO_C%2B%2B_Logo.svg/1200px-ISO_C%2B%2B_Logo.svg.png) uses Hue=215 as the major color. It would be better to use the same color for CC to manifest its connection to C++
- Using the complement color of `CC` for `LS`

I created a sample pallette for this here: https://paletton.com/#uid=23n0u0kJS6KvW5fuP8MNdeqCnl7

In short: CC=025568 LS=A85400

<img src="https://user-images.githubusercontent.com/8121231/85955128-9367e700-b94a-11ea-9904-3452a0817254.png" data-canonical-src="https://user-images.githubusercontent.com/8121231/85955128-9367e700-b94a-11ea-9904-3452a0817254.png" width="200" height="200" />

**Comment by mys721tx** [comment]
@htfy96 Hue=215 is a good choice. It looks too dark. Maybe CC=0056CD, LS=CD5100?

![ccls](https://user-images.githubusercontent.com/561975/85955351-d1591100-b932-11ea-9f3a-034823f4fd39.png)

**Comment by iromero91** [comment]
Chicago cubs language server!
![image](https://user-images.githubusercontent.com/8483301/89504129-2afff700-d78d-11ea-9896-c0b29adcd836.png)
